test/integration_test.exs

```diff
@@ -34,4 +35,15 @@ defmodule JokenJwks.IntegrationTest do
     assert signers = Strategy.EtsCache.get_signers()
     assert Enum.count(signers) >= 1
   end
+
+  test "can parse IdentityServer's JWKS" do
+    Strategy.start_link(
+      jwks_url: @application_jwks_set_certs_url,
```

Comment:
Perhaps you meant `@identity_server_certs_url`?

Reply:
Yes, last minute renames was apparently more than I could handle.
It should now be fixed.

Reply:
hmm... Travis CI is not happy. One elixir configuration fails with an error I can't quite understand, the other succeeds? I can run the test locally with same Elixir/Erlang versions?